cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -326,6 +326,7 @@ module Cardano.Api.Typed (
     toByronTxOut,
     toShelleyNetwork,
     toShelleyPoolParams,
+    toShelleyStakeAddr,
```

Comment:
Commit message could be improved. I have a good idea what all the words in the commit message mean, but I am still unsure what the commit actually does.

Reply:
Done.